test/CMakeLists.txt

```diff
@@ -73,6 +74,7 @@ if(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
     list(APPEND SRC tf_fake_input.cc)
     list(APPEND SRC test_ng_var_update_ng_tensor_kernel.cc)
     list(APPEND SRC graph_rewrites/test_ng_var_update_ng_tensor.cc)
+    add_definitions(-DNGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
```

Comment:
What is this for?

Reply:
So that we can #ifdef inside tests. 
https://github.com/tensorflow/ngraph-bridge/pull/334/files#diff-075ba2a5dfeeb28efa336a027e3f0964R43